.vale/styles/CheDocs/TechnicalTerms.yml

```diff
@@ -0,0 +1,29 @@
+---
+extends: substitution
+ignorecase: false
+level: error
+message: Use '%s' instead of '%s'
+scope: sentence
+swap:
+  '\sContainer Registry': Red Hat Ecosystem Catalog
+  '\sContainer Catalog': Red Hat Ecosystem Catalog
+  '\sconfig map': ConfigMap
+  '\sconfig maps': ConfigMaps
+  '\sDotnet': .NET
+  '\sdotnet': .NET
+  '\sgithub': GitHub
+  '\side': IDE
+  '\sjetbrains': Jetbrains
+  '\sjava': Java
+  '\netcoredebugoutput': NetcoredebugOutput
+  '\soperator': Operator
+  '\spod': Pod
```

Comment:
Do we capitalize it to match Kubernetes' documentation?
OpenShift has given up the k8s-style capitalization (they only capitalize Operators now).

Reply:
I guess so.